edge-modules/edgehub-proxy/Cargo.toml

```diff
@@ -7,17 +7,18 @@ edition = "2018"
 [dependencies]
 clap = "2.31"
 env_logger = "0.5"
-failure = "0.1"
-failure_derive = "0.1"
-hyper = "0.12.36"
+hyper = "0.14"
+thiserror = "1"
 log = "0.4"
 serde_json = "1.0"
-tokio = "0.1"
+tokio = { version = "1", features = ["full"] }
```

Comment:
Do we need 'full' or is there a small subset that is required?

Reply:
Only runtime is needed. I had brought in the full set of features earlier when I was figuring tokio out and didn't change it back. Changed